test/e2e/autoscaling/horizontal_pod_autoscaling.go

```diff
@@ -229,14 +229,14 @@ func scaleOnIdleSideCar(name string, kind schema.GroupVersionKind, checkStabilit
 	}
 	scaleTest := &HPAContainerResourceScaleTest{
 		initPods:                    1,
-		totalInitialCPUUsage:        250,
-		perContainerCPURequest:      500,
+		totalInitialCPUUsage:        125,
+		perContainerCPURequest:      250,
 		targetCPUUtilizationPercent: 20,
 		minPods:                     1,
 		maxPods:                     5,
 		firstScale:                  3,
 		firstScaleStasis:            stasis,
-		cpuBurst:                    700,
+		cpuBurst:                    500,
```

Comment:
Why every other value was divided by 2 and this one was changed differently?
(I understand it works anyway cause we set the max pods - I want to understand why "scaling" the test was not an option)

Reply:
maybe I miss understood this value, I though it was the amount of cpu put on to the pods to force scaling the pods.  I thought as long as this value is above the cpu request it would cause scaling.  Leaving it at 700 would work as well but I though should dial back to keep in line with other values.

> I want to understand why "scaling" the test was not an option

I don't understand what you mean by "scaling" the test?

Reply:
I confirmed this value is the amount of cpu to consume on the container: https://github.com/kubernetes/kubernetes/blob/f5e97186d297ee4d6254f634a591234ccdbe5ecb/test/e2e/autoscaling/horizontal_pod_autoscaling.go#L218

Reply:
I thought that burst number was calculated to trigger exactly 5 replicas (even if max pods number were higher). But it seems that it was previously adjusted for 7 replicas which is above the limit. My bad.

By "scaling" the test I meant diving each cpu value by 2, so the ratio will be conserved.